.gitignore

```diff
@@ -38,4 +38,5 @@ build/
 *.DS_Store
 *.pl
 *.a
-.vscode/
\ No newline at end of file
+.vscode/
+*.ss[0-9]
```

Comment:
```suggestion

```

Reply:
These are mGBA save state files, which I was using to test the built ROM. Is there an issue with ignoring these?

Reply:
Oh right. I barely use save states so that's why I didn't recognize the extension. I'm not too opposed to ignoring them.